tests/UserTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use App\User;
+
+class UserTest extends IntegrationTestCase
+{
+    /** @test */
+    function it_checks_if_user_is_admin()
+    {
+        $user = factory(User::class)->create();
+        $this->assertEquals(0, $user->role);
+        $this->assertFalse($user->isAdmin());
+
+        $admin = factory(User::class)->create(['role' => 1]);
+        $this->assertEquals(1, $admin->role);
+        $this->assertTrue($admin->isAdmin());
+    }
+
+    /** @test */
+    function it_returns_all_users_subscribed_to_notifications()
+    {
+        factory(User::class)->create();
+        factory(User::class)->create(['enable_notifications' => true]);
+
+        $this->assertEquals(1, User::EnabledNotifications()->count());
+    }
+
```

Comment:
delete empty line please

Reply:
Maybe we can use a style checker? Or do you maybe got PhpStorm project style settings you could share? I am so used to my auto format in PhpStorm that I accidentally mess up other code of the styles are different. Sry for that.

Reply:
https://github.com/tightenco/tlint

I think the TLint PHPStorm plugin should catch most of it--no worries at all! :)

Reply:
Thanks, I will try it before the next commit 👍